### PR TITLE
Event/2023orsal

### PR DESCRIPTION
**Pull request by rutmanz**

### CODE REVIEW

File: src/main/java/org/team1540/robot2023/Constants.java [hunk 1]
```diff
@@ -170,10 +170,15 @@ public static final class Mod3 {
...
+        public static final double kP = -0.03;
+        public static final double kI = 0;
+        public static final double kD = -0.002;
+    }
 
     public static final class ArmConstants {
-        public static final int PIVOT1_ID = 10;
-        public static final int PIVOT2_ID = 11;
+        public static final int PIVOT1_ID = 11;
+        public static final int PIVOT2_ID = 10;
```
Comment (mimizh2418) [inline src/main/java/org/team1540/robot2023/Constants.java:181]: swap these back around

File: src/main/java/org/team1540/robot2023/commands/arm/Arm.java [hunk 2]
```diff
@@ -47,11 +47,12 @@ public Arm() {
         pivot2.setNeutralMode(NeutralMode.Brake);
         telescope.setIdleMode(CANSparkMax.IdleMode.kBrake);
 
-        pivot1.setInverted(true);
+        pivot1.setInverted(false);
```
Comment (mimizh2418) [inline src/main/java/org/team1540/robot2023/commands/arm/Arm.java:50]: invert these again

**Review by mimizh2418 — CHANGES_REQUESTED** [review]
Fix the pivot constants and inversions because the belt is on now. Otherwise looks good.

**Comment by rutmanz** [comment]
> Fix the pivot constants and inversions because the belt is on now. Otherwise looks good.

Does it actually matter? Ideally we make a way at some point to swap them mid-match, but I can't see a reason that one configuration would be preferred over the other

**Comment by mimizh2418** [comment]
> > Fix the pivot constants and inversions because the belt is on now. Otherwise looks good.
> 
> Does it actually matter? Ideally we make a way at some point to swap them mid-match, but I can't see a reason that one configuration would be preferred over the other

I guess it doesn't matter, I just find it unsettling. I do agree that we should at some point make a button that switches the drive motor though.

**Review by mimizh2418 — APPROVED** [review]

**Pull request closed without merging**